client/components/forms/form-select/index.jsx

```diff
@@ -1,19 +1,28 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classnames = require( 'classnames' ),
-	omit = require( 'lodash/omit' );
+import React from 'react';
+import classNames from 'classnames';
+import omit from 'lodash/omit';
 
-module.exports = React.createClass( {
+const FormSelect = React.createClass( {
+	getDefaultProps() {
+		return {
+			isError: false,
+		};
+	},
```

Comment:
On second thought - not sure what "all props" are here :/ Part of the implementation seems to be that we allow arbitrary props in - same as with `form-text-input`.